functions/stencils_numbaloop.py

```diff
@@ -5,10 +5,9 @@
 # ******************************************************
 
 import numpy as np
-from numba import jit#, njit
+from numba import jit  # , njit
+
 
-@jit(nopython=True, debug=True)
-# @njit()
```

Comment:
All `@jit` and `@njit` decorators should be removed from the modules in `functions/` so to move the stencil compilation in `stencil_main.py`. This allows to get rid of `stencils_numbajit.py` and avoids the need for a `stencils_cupy..py` should we decide to run stuff on GPUs as well.